jsonschema2pojo-gradle-plugin/src/main/groovy/org/jsonschema2pojo/gradle/GenerateJsonSchemaAndroidTask.groovy

```diff
@@ -57,4 +59,12 @@ class GenerateJsonSchemaAndroidTask extends SourceTask {
 
     Jsonschema2Pojo.generate(configuration, new GradleRuleLogger(logger))
   }
+
+  void setTargetVersion(JsonSchemaExtension configuration) {
+    if (!configuration.targetVersion) {
+      def compileJavaTask = project.getTasksByName("compileJava", false).first()
+      configuration.targetVersion = compileJavaTask.getProperties().get("sourceCompatibility")
```

Comment:
should this be `targetCompatibility` ?
seems to disagree with the log output below

same problem in other file

Reply:
Ah yes, well spotted, thanks. I will update the log message. I changed my mind about which property was more appropriate, but didn't update the log message.